lib/ci-stack.ts

```diff
@@ -161,6 +161,7 @@ export class CIStack extends Stack {
         agentNode.AL2_ARM64_DOCKER_HOST,
         agentNode.AL2023_ARM64_DOCKER_HOST,
         agentNode.AL2023_X64_DOCKER_HOST_BENCHMARK_TEST,
+        agentNode.AL2023_X64_DOCKER_HOST_BENCHMARK_TEST_NEW_SPEC,
```

Comment:
I think this is a good opportunity to remote "DOCKER" from the agent label as we run everything on the localhost and not inside a CI docker container. @peterzhuamazon

Reply:
Will change, thanks!